frontstage/common/session.py

```diff
@@ -80,6 +80,11 @@ def save(self):
         redis.setex(self.session_key, 3600, self.encoded_jwt_token)
         self.persisted = True
 
+    def get_formatted_expires_in(self, refresh=None):
+        if refresh:
+            self.refresh_session()
```

Comment:
We shouldn't need to do this, @jwt_authorization(request) extends the session https://github.com/ONSdigital/ras-frontstage/blob/main/frontstage/common/authorisation.py#L62 so the views you have updated below will actually do it twice. As mentioned in another comment if you had 2 methods, one for GET and one for PATCH which uses the 2 refresh states in jwt_authorization, (default and false) everything should work without doing much else